.tekton/pull-request.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             - name: workspace
           steps:
             - name: run-tests
-              image: registry.access.redhat.com/ubi8/ubi:latest
+              image: registry.access.redhat.com/ubi9/python-39:latest
```

Comment:
We should not change this image. It won't make a difference. Our changes should not affect this step at all - not on this yaml and not on the makefile as far as I can see it.

Reply:
> Nice progress!

Thank you for your inputs.